enums.h

```diff
@@ -253,6 +253,10 @@ typedef enum _sfx_id {
 	TSFX_SMITH56,
 	TSFX_COW1,
 	TSFX_COW2,
+	TSFX_COW3,
+	TSFX_COW4,
+	TSFX_COW5,
+	TSFX_COW6,

```

Comment:
These have to match the order of the texts or you will have wrong sounds playing in-game...